src/sidebar/services/groups.js

```diff
@@ -154,8 +154,9 @@ function groups(
         documentUri = uri;
 
         if (features.flagEnabled('community_groups')) {
+          params.expand = ['organization', 'scopes'];
```

Comment:
I'm not sure this is strictly necessary since all the groups returned from this endpoint should already be scoped to this uri?

Reply:
Currently this is the case. Getting `scope` info in the response for all groups might be convenient in later code as you'll be able to treat all group objects the same way. See my comment in the `isSelectable` implementation about handling of groups where the scope info is missing though.

Reply:
+1 on the shape of all retrieved groups being identical.